tests/modules/elasticsearch_helper_index_management_test/elasticsearch_helper_index_management_test.info.yml

```diff
@@ -1,5 +1,6 @@
 name: Elasticsearch helper index management test
 type: module
 description: Module for Elasticsearch Helper Index management tests
+core_version_requirement: ^8 || ^9
 core: 8.x
-package: Testing
\ No newline at end of file
+package: Testing
```

Comment:
I think module package should be `ElasticSearch Helper` to match the package for other Elasticsearch Helper family modules (e.g., https://github.com/wunderio/elasticsearch_helper_content/blob/8.x-7.x/elasticsearch_helper_content.info.yml#L5).

Reply:
Package definitions added to info.yml files